src/app/rundown-view/components/rundown/rundown.component.ts

```diff
@@ -166,4 +167,8 @@ export class RundownComponent implements OnInit, OnDestroy, OnChanges {
     this.rundownEventSubscriptions.forEach(subscription => subscription.unsubscribe)
     this.rundownActionsSubscription?.unsubscribe()
   }
+
+  public isRundownActive(): boolean {
+    return [RundownMode.ACTIVE, RundownMode.REHEARSAL].includes(this.rundown.mode)
+  }
```

Comment:
Instead of evaluating this on every render call, it can be computed in `ngOnChange` and be cached.

Reply:
I have changed it to only be updated in `ngOnChanges`.

Ideally, however, we should not have this attribute in this component at all. It is only used to be parsed down through _3_ components before it is being used. The components that wants to actually use this value should just get the information from the RundownStateService.
But doing that is a refactor a bit out of scope of the intention of this PR.